data/icons.gresource.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<gresources>
+  <gresource prefix="/io/elementary/switchboard/plug/security-privacy">
```

Comment:
Same as https://github.com/elementary/onboarding/pull/208/files#r1213444764 if we namespace this to the application ID, it automatically gets included in the icons path
```suggestion
  <gresource prefix="io/elementary/switchboard/icons">
```

Reply:
This is cool, I did not know about this feature!